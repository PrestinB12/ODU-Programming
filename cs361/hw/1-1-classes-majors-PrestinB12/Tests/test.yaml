test:
  timelimit: 10
  launch: ./listMajor
  stderr: false
dueDate: 09/07/2023
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["readingList.h", "readingList.cpp"]
  instructorFiles:
    exclude: ["readingList.h", "readingList.cpp"]
