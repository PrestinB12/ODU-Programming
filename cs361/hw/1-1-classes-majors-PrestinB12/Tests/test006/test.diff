3d2
< **Memory leaks detected: 10
