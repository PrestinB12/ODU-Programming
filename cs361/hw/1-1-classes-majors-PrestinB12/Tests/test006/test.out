Semester 1
   math101 ()
**Memory leaks detected: 10
